client/src/components/GameArea.js

```diff
@@ -43,30 +61,286 @@ class GameArea extends Component {
     }
   }
 
+  increment(x) {
+    this.setState({
+      life: Number(this.state.life) + Number(x)
+    })
+  }
+
+  decrement(x) {
+    this.setState({
+      life: Number(this.state.life) - Number(x)
+    })
+  }
+
+  addCardToTopBattlefield(name, cost, img, type, set, text, power, toughness) {
```

Comment:
Nothing to change but will this method eventually be sending actions / updating state?

Reply:
I was thinking this method just be kept separate for rendering Cards.